application/aam-integration/src/main/kotlin/com/aamdigital/aamintegration/authentication/repository/AuthenticationSessionEntity.kt

```diff
@@ -0,0 +1,49 @@
+package com.aamdigital.aamintegration.authentication.repository
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import org.hibernate.annotations.CreationTimestamp
+import org.hibernate.annotations.SourceType
+import org.hibernate.annotations.UpdateTimestamp
+import java.time.OffsetDateTime
+
+@Entity
+data class AuthenticationSessionEntity(
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    val id: Long = 0,
+
+    @Column(unique = true)
+    var externalIdentifier: String,
+
+    @Column
+    var userId: String,
+
+    @Column
+    var externalUserId: String,
```

Comment:
could you add code comments here, explaining these briefly? (what's the difference between these three?)

Reply:
externalIdentifier: uuid for use to external API

userId: keycloak userId in our system

externalUserId: (see API docs) id in Tola